diff --git a/gfx/angle/targets/libEGL/moz.build b/gfx/angle/targets/libEGL/moz.build
index c16b620..99c8adf 100644
--- a/gfx/angle/targets/libEGL/moz.build
+++ b/gfx/angle/targets/libEGL/moz.build
@@ -6,7 +6,7 @@ include('../../moz.build.common')
 DEFINES['ANGLE_ENABLE_DEBUG_ANNOTATIONS'] = True
 DEFINES['ANGLE_GLESV2_LIBRARY_NAME'] = '"libGLESv2"'
 #DEFINES['ANGLE_IS_64_BIT_CPU'] = True
-DEFINES['ANGLE_USE_EGL_LOADER'] = True
+#DEFINES['ANGLE_USE_EGL_LOADER'] = True
 #DEFINES['CERT_CHAIN_PARA_HAS_EXTRA_FIELDS'] = True
 #DEFINES['CHROMIUM_BUILD'] = True
 #DEFINES['COMPONENT_BUILD'] = True
@@ -179,4 +179,4 @@ OS_LIBS += [
 
 DEFFILE = '../../checkout/src/libEGL/libEGL.def'
 RCFILE = '../../checkout/src/libEGL/libEGL.rc'
-GeckoSharedLibrary('libEGL', linkage=None)
\ No newline at end of file
+GeckoSharedLibrary('libEGL', linkage=None)
diff --git a/gfx/angle/update-angle.py b/gfx/angle/update-angle.py
index a1e677d..1e0df7e 100644
--- a/gfx/angle/update-angle.py
+++ b/gfx/angle/update-angle.py
@@ -353,7 +353,6 @@ REGISTERED_DEFINES = {
     'ANGLE_GLESV2_LIBRARY_NAME': True,
     'ANGLE_IS_64_BIT_CPU': False,
     'ANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES': False,
-    'ANGLE_USE_EGL_LOADER': True,
     'CERT_CHAIN_PARA_HAS_EXTRA_FIELDS': False,
     'CHROMIUM_BUILD': False,
     'COMPONENT_BUILD': False,
