diff --git a/gfx/angle/targets/libEGL/moz.build b/gfx/angle/targets/libEGL/moz.build
index 9327db4..390eb73 100644
--- a/gfx/angle/targets/libEGL/moz.build
+++ b/gfx/angle/targets/libEGL/moz.build
@@ -7,7 +7,7 @@ DEFINES["ANGLE_DISABLE_POOL_ALLOC"] = True
 DEFINES["ANGLE_ENABLE_SHARE_CONTEXT_LOCK"] = "1"
 DEFINES["ANGLE_GLESV2_LIBRARY_NAME"] = '"libGLESv2"'
 # DEFINES["ANGLE_IS_WIN"] = True
-DEFINES["ANGLE_USE_EGL_LOADER"] = True
+# DEFINES["ANGLE_USE_EGL_LOADER"] = True
 DEFINES["ANGLE_VMA_VERSION"] = "2003000"
 # DEFINES["CERT_CHAIN_PARA_HAS_EXTRA_FIELDS"] = True
 DEFINES["CR_CLANG_REVISION"] = '"llvmorg-16-init-6578-g0d30e92f-2"'
diff --git a/gfx/angle/update-angle.py b/gfx/angle/update-angle.py
index 57714e4..0b78c3b 100755
--- a/gfx/angle/update-angle.py
+++ b/gfx/angle/update-angle.py
@@ -270,7 +270,6 @@ REGISTERED_DEFINES = {
     "ANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES": False,
     "ANGLE_SHARED_LIBVULKAN": True,
     "ANGLE_USE_CUSTOM_LIBVULKAN": True,
-    "ANGLE_USE_EGL_LOADER": True,
     "ANGLE_VK_LAYERS_DIR": True,
     "ANGLE_VK_MOCK_ICD_JSON": True,
     "ANGLE_VMA_VERSION": True,
